contentcuration/contentcuration/viewsets/base.py

```diff
@@ -659,7 +660,12 @@ def create_from_changes(self, changes):
     def create(self, request, *args, **kwargs):
         serializer = self.get_serializer(data=request.data)
         serializer.is_valid(raise_exception=True)
-        self.perform_create(serializer)
+
+        try:
+            self.perform_create(serializer)
+
+        except IntegrityError as e:
+            return Response({"error": str(e)}, status=409)
```

Comment:
I think our main objective while implementing this was to try and throw a more meaningful error when creating a channel that the Id already exists.

Reply:
Okay! I was simply thinking about whether the change was as meaningful to the end-user as it is to us, since it would silence error reports on our end.

Reply:
I agree!  But again, do we need a redirect when double creation happens? Or we could try and find a way of letting the user know if double creation happened.

Reply:
I think it's very unlikely a user encounters this scenario submitting the form. But if they somehow did encounter it, or perhaps even a gateway timeout 502 error, I don't think the frontend would give the user notice one of those errors occurred, meaning the submit button would be forever in a disabled without any indication to the user.